### DISCUSSION TITLE
Looking for these few Portuguese channels.

### DISCUSSION THREAD

**Discussion by NettieJ85** [iptv-org/community#1070] [Q&A]
Any links for Trace Toca, Afro Music Pop, and/or Sic Mulher? Thank you.

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
